README.md

```diff
@@ -65,21 +65,43 @@ Cloning and Startup
 
   ``git clone https://github.com/openedx/frontend-app-gradebook.git``
 
-2. Install npm dependencies:
+2. Use node v20.x. The current version of the micro-frontend build scripts
+   supports node 20. Using other major versions of node may work, but this is
+   unsupported. For convenience, this repository includes an .nvmrc file to help
+   in setting the correct node version via [nvm](https://github.com/nvm-sh/nvm).
```

Comment:
I worry that specifically mentioning node 20 here could lead to this being outdated when node is upgraded. I'd rather just say "use the version in `.nvmrc`" so we can use that as a single source of truth.

Reply:
OK. When I'm in fea-authoring, would you like me to update the note there? (I cribbed from that README which says this exactly)

Reply:
That'd be awesome! I know I will absolutely forget to update `README`s when upgrading MFEs to a new version of node, but will definitely not forget to update `.nvmrc` files.